app/policies/collection_policy.rb

```diff
@@ -32,5 +33,10 @@ def deposit?
     NO_DEPOSIT_STATUS.exclude? record.state
   end
 
+  sig { returns(T::Boolean) }
+  def destroy?
+    (administrator? || collection_creator?) && record.state == 'first_draft' && record.works.count.zero?
```

Comment:
A draft collection will not have any works, so no need to check for that.
```suggestion
    (administrator? || collection_creator?) && record.first_draft?
```

Reply:
huh, interesting -- I had some draft works in a draft collection on my localhost from previous testing, which is the reason I added this condition.  is that not true?  I mean, we can allow the draft works inside of the draft collection to be destroyed via `has_many :works, dependent: :destroy`, but I just thought it would be safer to not allow that in the UI

Reply:
We have added restrictions that prevents works from being created in draft colletions recently: https://github.com/sul-dlss/happy-heron/blob/main/app/policies/work_policy.rb#L19

Reply:
got it, thanks